zerver/tests/test_integrations.py

```diff
@@ -41,33 +40,36 @@ def test_get_bot_avatar_path(self) -> None:
         self.assertIsNone(integration.get_bot_avatar_path())
 
     def test_no_missing_doc_screenshot_config(self) -> None:
-        webhook_names = {webhook.name for webhook in WEBHOOK_INTEGRATIONS}
-        webhooks_with_screenshot_config = set(WEBHOOK_SCREENSHOT_CONFIG.keys())
-        missing_webhooks = webhook_names - webhooks_with_screenshot_config - NO_SCREENSHOT_WEBHOOKS
-        extra_webhook_configs = webhooks_with_screenshot_config - webhook_names
-        extra_webhook_no_configs = NO_SCREENSHOT_WEBHOOKS - webhook_names
+        integration_names = {integration.name for integration in INTEGRATIONS.values()}
+        integrations_with_screenshot_configs = set(DOC_SCREENSHOT_CONFIG.keys())
+
+        missing_integrations = (
+            integration_names - integrations_with_screenshot_configs - NO_SCREENSHOT_CONFIG
+        )
+        extra_integration_configs = integrations_with_screenshot_configs - integration_names
+        extra_integration_no_configs = NO_SCREENSHOT_CONFIG - integration_names
 
         def construct_message(title: str, integrations: set[str], action: str) -> str:
             return (
                 f"\n\n{title}\n" + "\n".join(integrations) + f"\n{action}" if integrations else ""
             )
 
         self.assertEqual(
-            webhooks_with_screenshot_config,
-            webhook_names - NO_SCREENSHOT_WEBHOOKS,
+            integrations_with_screenshot_configs,
+            integration_names - NO_SCREENSHOT_CONFIG,
             construct_message(
                 "The following integrations are missing their example screenshot configuration:",
-                missing_webhooks,
+                missing_integrations,
```

Comment:
Maybe rename to `missing_integration_screenshot` or something like that for a bit more clarity.